src/pluto/grey_image.cr

```diff
@@ -0,0 +1,76 @@
+require "./bindings/*"
+require "./format/*"
+require "./operation/*"
+
+class Pluto::GreyImage
+  include Format::JPEG
+  include Format::PPM
+  include Format::Save
+  extend Format::Open
+
+  include Operation::BilinearResize
+  include Operation::BoxBlur
+  include Operation::Brightness
+  include Operation::ChannelSwap
+  include Operation::Contrast
+  include Operation::GaussianBlur
+  include Operation::HorizontalBlur
+  include Operation::VerticalBlur
+
+  getter grey : Array(UInt8)
+  getter width : Int32
+  getter height : Int32
+
+  def self.new(red : Array(UInt8), green : Array(UInt8), blue : Array(UInt8), alpha : Array(UInt8), width : Int32, height : Int32)
+    Image.new(red, green, blue, alpha, width, height).to_grey
+  end
+
+  def initialize(@grey, @width, @height)
+  end
+
+  def clone : GreyImage
+    self.class.new(
+      @grey.clone,
+      @width,
+      @height
+    )
+  end
+
+  def red : Array(UInt8)
+    @grey
+  end
+
+  def green : Array(UInt8)
+    @grey
+  end
+
+  def blue : Array(UInt8)
+    @grey
+  end
```

Comment:
These methods help make grey images completely compatible with regular color images (for the sake of saving).